.travis.yml

```diff
@@ -256,6 +256,10 @@ matrix:
     - php: 7.1
       env: DB=pgsql POSTGRESQL_VERSION=9.6
 
+before_install:
+  # Force hhvm PHP 7 mode
+  - if [[ $TRAVIS_PHP_VERSION = hhv* ]]; then echo hhvm.php7.all=1 >> /etc/hhvm/php.ini; fi
```

Comment:
Yes hhvm or like hhvm-3.12 
I think I may have tried hhvm* when I first got Travis to support specifying different versions of hhvm but only hhv* worked.